src/router.js

```diff
@@ -37,6 +37,14 @@ const logoutAndRedirect = async (to, from, next) => {
   return next('/login');
 };
 
+const redirectIfNoIdParam = (to, from, next) => {
+  if (to.params.gameId || to.params.seasonId) {
+    next();
+  } else {
+    next('/');
```

Comment:
This is the right idea, but the reason this won't work is that the first time you navigate to the /stats page, we don't have the ids of any of the seasons yet (the stats view downloads them on created).